dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyTreeFormatter.java

```diff
@@ -98,14 +98,15 @@ private static void formatDependencyPathTree(
   }
 
   /**
-   * Builds ListMultiMap that represents a Maven dependency tree of parent-children relationship.
-   * The root node is represented as an empty {@link DependencyPath} and its children are
-   * the values for the root node.
+   * Builds ListMultimap that represents a Maven dependency tree of parent-children relationship.
```

Comment:
Yes, but how? Now that this is a public method this needs to be described in sufficient detail for someone to understand the representation; that is, how a multimap turns into a tree, or how a tree is encoded in a multimap. They are not the same data structure, so how are we using this here? That's what needs to be clearer.

Reply:
To be more specific, I think you need to say something like:

Each node in the tree is a key in the multimap. The multiple values of that key are the children of that node.

Reply:
That's good explanation. Updated.